Helper/Settings.php

```diff
@@ -314,4 +314,15 @@ public function getHelperByTemplateId($templateId)
 
         return null;
     }
+
+    /**
+     * To get magento version
+     *
+     * @return string
+     */
+    public function getMagentoVersion()
+    {
+        $productMetadata = $this->objectManager->get('Magento\Framework\App\ProductMetadataInterface');
```

Comment:
@ladonenko do not use OM here, inject ProductMetadataInterface via construct and use it here

Reply:
added ProductMetadataInterface to DI